.github/workflows/ci.yml

```
@@ -28,17 +28,16 @@ jobs:
       run: |
         luarocks install luafilesystem
         luarocks install luacov
-        luarocks install luacov-coveralls --server=http://rocks.moonscript.org/dev
+        git clone https://github.com/alerque/luacov-coveralls.git --branch fix-actions-service-name
+        luarocks install ./luacov-coveralls/rockspecs/luacov-coveralls-scm-0.rockspec
```

...and use the rockspec we just fetched:

```suggestion
        luarocks install ./luacov-coveralls-TEST-0.rockspec
```